docker/php/Dockerfile

```diff
@@ -19,7 +19,10 @@ RUN composer --version
 RUN docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/
 
 # Install PHP extensions
-RUN docker-php-ext-install pdo pdo_mysql zip gd intl
+RUN docker-php-ext-install pdo mysqli pdo_mysql zip gd intl
+
+# Enable PHP Extensions
+RUN docker-php-ext-enable mysqli
```

Comment:
This line is not necessary, extension is enabled on install, can you remove it ? I will merge your request then